addons/web_editor/static/src/xml/commands.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates>
+    <t t-name="web_editor.banner">
+        <div t-attf-class="o_we_banner_container lh-1 d-flex alert alert-#{alertClass}" role="status" data-oe-protected="true" contenteditable="false">
+            <i t-attf-class="fa #{iconClass}" t-att-aria-label="label"/>
+            <div class="o_we_banner_editable w-100 ms-3 oe_unremovable" data-oe-protected="false"
+                t-att-contenteditable="!readonly ? 'true' : 'false'">
```

Comment:
Same here IMO. I'd just use the `o_editable` class. If the widget is in readonly mode then the editor will not deduce `contentEditable` from it, which is what you want I think.